server/restapi/migrations/0001_initial.py

```diff
@@ -0,0 +1,23 @@
+# Generated by Django 3.0.6 on 2020-05-08 16:34
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    initial = True
+
+    dependencies = [
+    ]
+
+    operations = [
+        migrations.CreateModel(
+            name='Datauser',
+            fields=[
+                ('id', models.AutoField(auto_created=True, primary_key=True, serialize=False, verbose_name='ID')),
+                ('global_id', models.UUIDField(verbose_name='id')),
+                ('name_user', models.CharField(max_length=20, verbose_name='name')),
+                ('create_date', models.DateTimeField(verbose_name='data')),
```

Comment:
```suggestion
                ('created_at', models.DateTimeField(verbose_name='created_at')),
```

`data` не совсем равна `date` поэтому желательно их синхронизировать. Я предлагаю использовать стандарт именования дефакто в разных системах для времени -- `created_at` и `updated_at`. Также некоторые модули для django ориентируются по-умолчанию на эти поля.